src/ui/components/Notice/styles.scss

```diff
@@ -76,6 +76,31 @@
   }
 }
 
+.Notice-generic {
+  .Notice-button {
+    @include button(
+      $background: $blue-60,
```

Comment:
Hmm. I don't see blue anywhere in the [action button spec](https://design.firefox.com/photon/components/message-bars.html#call-to-action). Also, does it have the correct hover styles? That mixin is a bit confusing. You may need some overrides similar to the `success` one.

Reply:
It's in the spec @pwalm linked to in #4547: https://design.firefox.com/photon/components/message-bars.html#color

Reply:
Oh man, I'm so confused. I see the blue now in what you linked to but it's gray everywhere else.

The call to action part specifies gray values: https://design.firefox.com/photon/components/message-bars.html#call-to-action

The default button looks gray: https://design.firefox.com/photon/components/buttons.html#default-4